pom.xml

```diff
@@ -35,6 +35,8 @@
 
 	<properties>
 		<openapi-codegen-version>5.1.0</openapi-codegen-version>
+    <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
this comes in Via the Tile, so it isn't necessary. if you do a mvn help:effective-pom you will see the version of Java is defined there. It will be your IDE that is having trouble I expect?

Reply:
hmm. I'm not very familiar with maven, the last time I used anything related to java was some 3 years ago, which was an android app written in kotlin and using gradle and before that introductory uni courses :p And I never really had to deal with the build system.
But I set up emacs using the eclipse jdt language server and when I opened the file I got a whole bunch of errors like "the diamond operator <> is not allowed for source value 1.7"

Reply:
yeah, thats because eclipse doesn't support maven properly, lets leave them in - there is no harm :-)